src/gatsby-theme-docz/wrapper.js

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet-async';
+
+const Wrapper = ({ children }) => (
+  <React.Fragment>
+    <Helmet>
+      <meta charSet="utf-8" />
+      <link rel="icon" type="image/png" href="https://i.imgur.com/LdzriR3.png" />
+    </Helmet>
+    {children}
+  </React.Fragment>
```

Comment:
This is used to add a favicon (Gatsby automatically picks up this file if named in this way)